README.md

```diff
@@ -1,4 +1,54 @@
-rednoi
-======
+# RedNoi
 
-A Symfony project created on February 22, 2017, 7:06 pm.
+Overwhelmed? Let RedNoi helps you to overcome your information overload. 
+
+This tool is for advanced Twitter/TweetDeck users only, beginners might need to understand what Twitter/TweetDeck parameters are and how they work before using it.
+
+Rednoi uses possibilities of [The Search API][] where you can use the filters on public twitter lists. The results are then combined and serialized by time into one stream. So the final result provides the one data stream same like the Twitter and with filtered data of your specific lists.
+
+You can set very specific levels of a filtering for a list:
+- **min_retweets** ... minimum of retweets
+- **min_faves** ... minimum of favourites
+- **min_replies** ... minimum of replies
+- **-filter:replies** ... a tweet is not a replay
+- **lang:en** ... only in English
+- **(Trump OR US) AND Kim** ... logic operators
+
+Of course you can use it without a list (e.g.: *trump AND kim lang:en min_retweets:1000 min_faves:100*). 
+You can use the search query from TweetDeck. For more information please check [The Search API][].
+
+The best results are achieved when you create specific lists for each reducing levels. Suppose We follow 3 different twitter users: our good friend, our favourite tech influencer and source of globals news. These users are totally different. Therefore It would be better to create 3 different lists.
```

Comment:
for each reducing levels ➡️ for each of reducing levels (? - I don't understand much what "reducing level" is)